channeldb/payments.go

```diff
@@ -529,6 +529,23 @@ func (d *DB) QueryPayments(query PaymentsQuery) (PaymentsResponse, error) {
 			return fmt.Errorf("index bucket does not exist")
 		}
 
+		// Helper function to check if a payment meets the creation time
+		// criteria.
+		paymentInRange := func(payment *MPPayment) bool {
+			createTime := payment.Info.CreationTime.Unix()
+			if createTime < query.CreationDateStart {
```

Comment:
There shouldn't be an empty line after this `if` statement:
```
channeldb/payments.go:536: unnecessary leading newline (whitespace)
```

The extra line is only needed if the condition in the `if` statement spans over multiple lines, such as is the case in the next one. So the extra newline is a visual helper to show that a condition is multi-line.

Reply:
Understood. Thanks. Done.